CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+- The parser is stricter in a few cases where code that was incorrect (according to the Elm compiler) was successfully parsed: tuple patterns and type annotations with more than 3 elements, non-associative operators (so no more `a < b < c`), lambda indentation
```

Comment:
Did you mean "tuple patterns IN type annotations"? Because tuple expressions with more than 3 elements was already handled in the previous release. Or am I misunderstanding something?

Reply:
I meant the elm pattern for n-tuple and the elm type for n-tuple